index.html

```diff
@@ -830,21 +830,26 @@ <h1><a name="2NormRefs">2 Normative references</a></h1>
 3-900-734-00-3.</p>
 
 <p class="NormRefDef"><a name="2-ICC-1">ICC-1</a>, International
-Color Consortium, "Specification ICC.1: 1998-09, File Format for
-Color Profiles", 1998, available at <code><a href=
-"http://www.color.org/">http://www.color.org/</a></code></p>
-
-<p class="NormRefDef"><a name="2-ICC-1A">ICC-1A</a>,
-International Color Consortium, "Specification ICC.1A: 1999-04,
-Addendum 2 to ICC.1: 1998-09", 1999, available at <code><a href=
-"http://www.color.org/">http://www.color.org/</a></code></p>
+Color Consortium, "Specification ICC.1:2010-12 (Profile version 4.3.0.0)
+Image technology colour management - Architecture, profile format, and data structure,
+available at <code><a href=
+"http://www.color.org/icc_specs2.xalter">http://www.color.org/icc_specs2.xalter</a></code>
+also published as ISO 15076-1</p>
```

Comment:
This change has ICC-1 references (used in the iCCP & sRGB chunk definitions) pointing to the v4 ICC profiles.
But iCCP should remain v2 and the new iCCN should be v4?

I think the updated link is good. But it should probably say v2 instead of v4.

The sRGB chunk references it for the rendering intent. Maybe that is shared between v2 and v4? I'm not familiar enough here to know.